### ISSUE TITLE
UI crash in network view

### ISSUE THREAD

**Issue by jessesuen** [argoproj/argo-cd#1589]
```
react-dom.production.min.js:198 TypeError: Cannot read property 'x' of undefined
    at application-resource-tree.tsx:65
    at Array.forEach (<anonymous>)
    at application-resource-tree.tsx:64
    at On (application-resource-tree.tsx:353)
    at Zo (react-dom.production.min.js:150)
    at Yi (react-dom.production.min.js:178)
    at Ga (react-dom.production.min.js:232)
    at Xa (react-dom.production.min.js:233)
    at Os (react-dom.production.min.js:249)
    at Cs (react-dom.production.min.js:248)
```

**Comment by duboisf** [comment]
@jessesuen when does this happen? I've been hit with the same but not sure what triggered it and now it's working :thinking:

And this is with `1.0.0-rc2` I suppose?

**Comment by alexmt** [comment]
The error happens if you try to open application details page which we just created and not yet reconciled by the app controller. The fix is available in just released [v1.0.0-rc3](https://github.com/argoproj/argo-cd/releases/tag/v1.0.0-rc3)
